shared/components/StudyInfo/LearningPeriod.tsx

```diff
@@ -100,7 +88,8 @@ export class LearningPeriod extends React.Component<LearningPeriodProps> {
                     />
                   </TH>
                   <TD>
-                    {vastuullinenTyopaikkaOhjaaja.nimi}, {selite}
+                    {vastuullinenTyopaikkaOhjaaja.nimi}
+                    {!!selite ? `, ${selite}` : ""}
```

Comment:
{!!selite && `, ${selite}`} will also do fine

Reply:
I'll fix the underlying problem in a upcoming pr.